x/xcvm/types/codec.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	authzcodec "github.com/cosmos/cosmos-sdk/x/authz/codec"
+	govcodec "github.com/cosmos/cosmos-sdk/x/gov/codec"
+)
+
+var (
+	amino     = codec.NewLegacyAmino()
+	ModuleCdc = codec.NewAminoCodec(amino)
+)
+
+func init() {
+	RegisterLegacyAminoCodec(amino)
+	cryptocodec.RegisterCrypto(amino)
+
+	RegisterLegacyAminoCodec(authzcodec.Amino)
+	RegisterLegacyAminoCodec(govcodec.Amino)
+	amino.Seal()
```

Comment:
Why are these necessary?

Reply:
Good point - seems to just be a noop. I was basing the majority of this approach off of the `x/tx-boundary` module and it looks like it's a [noop there as well](https://github.com/ComposableFi/composable-cosmos/blob/release/v7.0.x/x/tx-boundary/types/codec.go). I'll remove this init section here for the `xcvm` module and remove the unused variables mentioned below.